docusaurus/docs/e2e-test-a-plugin/setup-resources.md

```diff
@@ -0,0 +1,83 @@
+---
+id: setup-resources
+title: Configure the resources you need
+description: Configure the resources you need
+keywords:
+  - grafana
+  - plugins
+  - plugin
+  - testing
+  - e2e
+  - data-source
+  - preparation
+sidebar_position: 60
+---
+
+# Introduction
```

Comment:
```suggestion
## Introduction
```
(otherwise the title of the page is omitted and you just see "introduction". May need to adjust subsequent headers as you feel appropriate.